src/main/kotlin/dev/stalla/builder/episode/EpisodeGoogleplayBuilder.kt

```diff
@@ -28,11 +29,13 @@ public interface EpisodeGoogleplayBuilder : Builder<EpisodeGoogleplay> {
     /** Set the [HrefOnlyImageBuilder]. */
     public fun imageBuilder(imageBuilder: HrefOnlyImageBuilder?): EpisodeGoogleplayBuilder
 
-    override fun applyFrom(prototype: EpisodeGoogleplay?): EpisodeGoogleplayBuilder = whenNotNull(prototype) { googleplay ->
-        author(googleplay.author)
-        description(googleplay.description)
-        explicit(googleplay.explicit?.type)
-        block(googleplay.block)
-        imageBuilder(HrefOnlyImage.builder().applyFrom(googleplay.image))
+    override fun applyFrom(prototype: EpisodeGoogleplay?): EpisodeGoogleplayBuilder {
+        return whenNotNull(prototype) { googleplay ->
+            author(googleplay.author)
+            description(googleplay.description)
+            explicit(googleplay.explicit?.type)
+            block(googleplay.block)
+            imageBuilder(HrefOnlyImage.builder().applyFrom(googleplay.image))
+        }
```

Comment:
Alternative here could be:

```
override fun applyFrom(prototype: EpisodeGoogleplay?): EpisodeGoogleplayBuilder =
  whenNotNull(prototype) { googleplay ->
    author(googleplay.author)
    description(googleplay.description)
    explicit(googleplay.explicit?.type)
    block(googleplay.block)
    imageBuilder(HrefOnlyImage.builder().applyFrom(googleplay.image))
  }
```

but that's not really a single line expression body anymore.

Reply:
It's not a single line but it it a single expression. Personal taste would prefer the expression body syntax, but easy either way

Reply:
I also prefer the single expression tbh. Just wanted to stick to the convention as closely as possible. But if you're also preferring it, I'll change it then.